packages/dds/merge-tree/src/properties.ts

```diff
@@ -96,7 +106,7 @@ export function matchProperties(a: PropertySet | undefined, b: PropertySet | und
 			}
 		}
 	} else {
-		if (b) {
+		if (b && entriesB.length > 0) {
```

Comment:
can we reuse entries b here

Reply:
I'm not sure what you mean here, is there a different comparison I can do that's cheaper?

Reply:
my bad. misread the diff